include/SimCore/LHEEvent.h

```diff
@@ -140,6 +146,12 @@ namespace ldmx {
              * Vertex location
              */
 	    double vtx_[3];
+
+
+        /**
+             * Vertex time
+             */
+        double vtxt_;
```

Comment:
Since `vtxt_` is not set in all running modes, it should have a sensible default here. I think `0.` makes the most sense.
```
double vtxt_{0.};
```